checker/src/org/checkerframework/checker/interning/InterningVisitor.java

```diff
@@ -240,6 +282,7 @@ private boolean overridesEquals(ClassTree node) {
      */
     private boolean isInvocationOfEquals(MethodInvocationTree node) {
         ExecutableElement method = TreeUtils.elementFromUse(node);
+        //TODO: CODE REVIEW NEITHER OF THE TWO
```

Comment:
This was supposed to read isInvocationOf methods will guarantee they are the right methods because you would also have to check the type of the parameter.  Perhaps the better way todo this would be to address the todo in TreeUtils.getMethod and then use that method.